static/js/containers/Payment.js

```diff
@@ -70,12 +124,38 @@ class Payment extends React.Component {
       <a href="#">Print your statement</a>
     </div>;
   }
+
+  render() {
+    const { selectedKlass } = this.props;
+
+    let renderedKlassDropdown, renderedSelectedKlass;
+    if (this.shouldShowKlassDropdown()) {
+      renderedKlassDropdown = this.renderKlassDropdown();
```

Comment:
why not just inline this (and the check right below) as ternarys in the JSX below?